### DISCUSSION TITLE
Is there a website like these for Connect for Chrome - Education extension?

### DISCUSSION THREAD

**Discussion by ugondia-2** [3kh0/ext-remover#390] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
![Screenshot 2023-01-10 3 17 51 PM](https://user-images.githubusercontent.com/100532913/211654586-2aa3e21c-5be8-4912-8c59-a205357766ea.png)

**Comment by Aka-but-nice** [comment]
Is that a blocker

**Reply by ugondia-2** [reply]
Yes.

**Reply by Aka-but-nice** [reply]
alr

**Reply by Aka-but-nice** [reply]
Well, it doesn't have XSS(Cross Site Scripting) so we can't make it work.

**Reply by ugondia-2** [reply]
It also goes by the name of LineWize.

**Comment by Alex2Mayhem** [comment]
> ![Screenshot 2023-01-10 3 17 51 PM](https://user-images.githubusercontent.com/100532913/211654586-2aa3e21c-5be8-4912-8c59-a205357766ea.png)

No there is not a way.

**Reply by Aka-but-nice** [reply]
Yea, because there's no XSS is Connect for chrome,

**Comment by rwg100** [comment, marked as answer]
Sure!

Step 1: drop 12 kg cinder block on top of chromebook

nice job! you have now successfully removed Connect for Chrome!

**Reply by ugondia-2** [reply]
Less destructive option?

**Reply by Aka-but-nice** [reply]
> Sure!
> 
> Step 1: drop 12 kg cinder block on top of chromebook
> 
> nice job! you have now successfully removed Connect for Chrome!

I really like this method, it's really inductive,
